webrtc/README.md

```diff
@@ -11,171 +11,27 @@ Interest Group: [@marten-seemann]
 [@marten-seemann]: https://github.com/marten-seemann
 [@mxinden]: https://github.com/mxinden/
 
-<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
-**Table of Contents**
+WebRTC flavors in libp2p:
 
-- [WebRTC](#webrtc)
-  - [Motivation](#motivation)
-  - [Addressing](#addressing)
-  - [Connection Establishment](#connection-establishment)
-    - [Browser to public Server](#browser-to-public-server)
-  - [Multiplexing](#multiplexing)
-    - [Ordering](#ordering)
-    - [Head-of-line blocking](#head-of-line-blocking)
-    - [`RTCDataChannel` negotiation](#rtcdatachannel-negotiation)
-    - [`RTCDataChannel` label](#rtcdatachannel-label)
-  - [Connection Security](#connection-security)
-  - [Previous, ongoing and related work](#previous-ongoing-and-related-work)
-  - [Test vectors](#test-vectors)
-    - [Noise prologue](#noise-prologue)
-      - [Both client and server use SHA-256](#both-client-and-server-use-sha-256)
-- [FAQ](#faq)
+1. [WebRTC](./webrtc.md)
 
-<!-- markdown-toc end -->
+   libp2p transport protocol enabling two private nodes (e.g. two browsers) to
+   establish a direct connection.
 
-## Motivation
+2. [WebRTC Direct](./webrtc-direct.md)
 
-1. **No need for trusted TLS certificates.** Enable browsers to connect to
-   public server nodes without those server nodes providing a TLS certificate
-   within the browser's trustchain. Note that we can not do this today with our
-   Websocket transport as the browser requires the remote to have a trusted TLS
-   certificate. Nor can we establish a plain TCP or QUIC connection from within
-   a browser. We can establish a WebTransport connection from the browser (see
-   [WebTransport specification](../webtransport)).
+   libp2p transport protocol **without the need for trusted TLS certificates.**
+   Enable browsers to connect to public server nodes without those server nodes
+   providing a TLS certificate within the browser's trustchain. Note that we can
+   not do this today with our Websocket transport as the browser requires the
+   remote to have a trusted TLS certificate. Nor can we establish a plain TCP or
+   QUIC connection from within a browser. We can establish a WebTransport
+   connection from the browser (see [WebTransport
+   specification](../webtransport)).
```

Comment:
It would be 82 characters long, thus the line wrap.

Note that the main `README.md` file is still structured with line wraps, not one-sentence-per-line as introduced in https://github.com/libp2p/specs/pull/412. I have deliberately not changed the formatting here to keep the diff small.